Tensile/Configs/navi21/rocblas_hgemm_gb_nn_asm_full.yaml

```diff
@@ -1,7 +1,7 @@
 # headers
 GlobalParameters:
   MinimumRequiredVersion: 4.9.0
-  PrintLevel: 1
```

Comment:
This was change to 2 because it was assumed that the code commands were desired but that may not be the case.